### PR TITLE
Add redis_data script for querying redis_events.beta table

### PR DESCRIPTION
**Pull request by yzhao1012**
.beta is an indicator of a beta feature.

### CODE REVIEW

File: pxl_scripts/px/redis_data/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+short: Redis RPC messages
+long: >
+  Traces Redis RPC request and response messages.
+  WARNING: Redis tracing is a beta feature. Details may subject to further changes.
+org_name: pixielabs.ai
```
Comment (oazizi000) [inline pxl_scripts/px/redis_data/manifest.yaml:6]: I don't think we are supposed to have an org name for our base pxl_scripts.

File: pxl_scripts/px/redis_data/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+short: Redis RPC messages
```
Comment (htroisi) [inline pxl_scripts/px/redis_data/manifest.yaml:2]: "long: Shows a sample of Redis messages in the cluster." would better match the descriptions of the other `px/*_data` scripts.

**Review by zasgar — APPROVED** [review]

File: pxl_scripts/px/redis_data/vis.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
+{
+  "variables": [
+    {
+      "name": "start",
+      "type": "PX_STRING",
+      "description": "The start time of the window in time units before now.",
+      "defaultValue": "-5m"
+    },
```
Comment (oazizi000) [inline pxl_scripts/px/redis_data/vis.json:8]: For consistency with other tables:

```
    {
      "name": "start_time",
      "type": "PX_STRING",
      "description": "The relative start time of the window. Current time is assumed to be now",
      "defaultValue": "-5m"
    },
```

File: pxl_scripts/px/redis_data/trace.pxl [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+''' Redis Data Tracer
+
+This script traces all Redis data on the cluster.
+'''
+
+import px
+
+
+def redis_data(start: str, num_head: int):
+    # redis_events.beta is a beta table, may subject to changes.
```
Comment (oazizi000) [inline pxl_scripts/px/redis_data/trace.pxl:13]: ...is a beta table, which is subject to change.

File: pxl_scripts/px/redis_data/manifest.yaml [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+long: >
+  Traces Redis RPC request and response messages.
+  WARNING: Redis tracing is a beta feature. Details may subject to further changes.
```
Comment (oazizi000) [inline pxl_scripts/px/redis_data/manifest.yaml:5]: I think the warning/note makes sense when loading the table inside the script (which you already have).

Here, maybe just say: "Redis tracing is a beta feature. This script is for demonstration purposes only."

**Pull request merged**